templates/bootstrap/features.tmpl

```diff
@@ -29,7 +29,7 @@
             <div class="panel panel-default">
               <div class="panel-heading">
                 <h4 class="panel-title">
-                <% var scenarioTagsOnly = feature.tags ? _.drop(element.tags, feature.tags.length) :  element.tags %>
+                <% var scenarioTagsOnly = element.tags %>
```

Comment:
Could we know the reasons for removing the tag filtering? If we remove this then it will pollute the report with duplicate tags at Scenario level. 

Please see below screenshots. 

* The `@unhappy @reporting` are part of the feature file so it should not be at Scenario level. We filter those tags to make it clean

<img width="584" alt="screen shot 2017-04-26 at 10 56 18 am" src="https://cloud.githubusercontent.com/assets/3663389/25441000/0ae3e0fe-2a6f-11e7-994e-e63f51de520e.png">


* With the filtering

<img width="586" alt="screen shot 2017-04-26 at 10 54 28 am" src="https://cloud.githubusercontent.com/assets/3663389/25441115/58cec414-2a6f-11e7-973d-e22a0ec6386f.png">